src/webparts/laserficheAdminConfiguration/components/ManageConfigurationsPage/ManageConfigurationsPage.tsx

```diff
@@ -86,19 +90,22 @@ export default function ManageConfigurationsPage(
     setDeleteModal(deleteModal);
   }
 
-  async function removeRowAsync(id: number) {
+  async function removeRowAsync(id: number): Promise<void> {
     const rows = [...configRows];
     const deleteRows = [...configRows];
     rows.splice(id, 1);
     await deleteMappingAsync(deleteRows, id);
     setDeleteModal(undefined);
   }
 
-  function closeModal() {
+  function closeModal(): void {
     setDeleteModal(undefined);
   }
 
-  async function deleteMappingAsync(rows: ProfileConfiguration[], idx: number) {
+  async function deleteMappingAsync(
+    rows: ProfileConfiguration[],
+    idx: number
+  ): Promise<void> {
     const manageConfigs: { id: string; configs: ProfileConfiguration[] } =
       await getManageConfigurationsAsync();
     if (manageConfigs.configs?.length > 0) {
```

Comment:
This change introduces a potential comparison between undefined and 0, but it's functionally equivalent. I think we may have seen this leading to a bug in the past, but if you think it's okay, I'm fine with it.